.github/workflows/node.js.yml

```diff
@@ -12,13 +12,17 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.ref }}
```

Comment:
I think for safety we should include the workflow name as well: https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow
Concurrency isn't used in any other workflows yet, but we do have other workflows